services/ui-src/src/components/fields/DateField.tsx

```diff
@@ -30,21 +26,15 @@ export const DateField = ({
 
   const [date, setDate] = useState("");
 
-  const onChangeHandler = async (event: string, formattedString: string) => {
-    setDate(event);
-    form.setValue(parentFieldName, event ? parseInt(event) : undefined, {
-      shouldValidate: true,
-    });
-    let year = parseInt(formattedString.split("/")?.[2]);
-    let month = parseInt(formattedString.split("/")?.[0]);
-    let day = parseInt(formattedString.split("/")?.[1]);
-    if (year && month && day) {
-      const time = calculateTimeByDateType(parentFieldName);
-      const calculatedDatetime = convertDateEtToUtc({ year, month, day }, time);
-      form.setValue(parentFieldName, calculatedDatetime, {
+  const onChangeHandler = async (inputtedString: string) => {
+    setDate(inputtedString);
+    form.setValue(
+      parentFieldName,
+      inputtedString ? inputtedString : undefined,
+      {
         shouldValidate: true,
-      });
-    }
+      }
+    );
   };
```

Comment:
I had to modify this handler a bit for our new validation stuff, so there will be some merge conflicts, but I don't think anything too bad.